src/sentry/integrations/slack/message_builder/__init__.py

```diff
@@ -0,0 +1,22 @@
+from typing import Any, Mapping
+
+# TODO(mgaeta) Flesh out this type.
+SlackBody = Mapping[str, Any]
+SlackBlock = Mapping[str, Any]
```

Comment:
do we actually use blocks? I thought we used attachments - this might be confusing if we do want to use blocks later on and "SlackBlock" actually means "SlackAttachment"

Reply:
`SlackDiscoverMessageBuilder` uses blocks. I've renamed the types.